### DISCUSSION TITLE
希望有一个开关，屏蔽动态页面的“话题”栏目

### DISCUSSION THREAD

**Discussion by Kki426** [the1812/Bilibili-Evolved#2262] [综合讨论 / General]
如图
![Snipaste_2021-09-09_12-46-15](https://user-images.githubusercontent.com/90371201/132624164-493c5106-0269-4527-8126-dbca47e43dcf.png)
这个栏经常会推荐一些广告性质的话题，比如最近b站做的综艺节目，或者是一些很无聊的话题，再或者是一些挑起对立的话题，观感很差。大佬如果有空的话，可以考虑做一个这方面的功能吗(￣▽￣)

**Comment by Geeyun-JY3** [comment]
有这功能的：https://github.com/the1812/Bilibili-Evolved/blob/master/doc/features.md#动态
在 `动态` 开启 `动态过滤器` 功能后，在动态首页相应设置面板开启对应选项：
![image](https://user-images.githubusercontent.com/49396421/132628570-f55c8eca-9653-4571-bab2-4e984ed34136.png)

**Reply by Kki426** [reply]
好的好的，感谢帮助，谢谢你
